application/dash_application/views/call_ready_wgs.py

```diff
@@ -282,7 +269,8 @@ def generate_deduplicated_coverage(df, graph_params):
         graph_params["colour_by"], 
         graph_params["shape_by"],
         graph_params["shownames_val"],
-        cutoff_lines=[(cutoff_coverage_tumour_label, graph_params[cutoff_coverage_tumour]), (cutoff_coverage_normal_label, graph_params[cutoff_coverage_normal])],
+        #TODO: Should these move to the CoverageMedian graph? 
```

Comment:
@slazicoicr ?

Reply:
Median vs mean is still up in the air (also true of Insert Size Mean/Median). I will bring it up at tomorrow's meeting.

Reply:
to clarify - this is median vs 'deduplicated', same deal?

Reply:
Still same deal. Median coverage cannot be easily deduplicated from current BamQC data, so if cutoff needs to be on median coverage deduplicated (and not mean coverage deduplicated as it currently is), more work is required. Still want confirmation.